### PR TITLE
UI Navigation Correction

### PR DESCRIPTION
**Pull request by leechristophermurray**
I changed some HTML attributes (eg. mislabelled classes) and refined the styling (borders and font colors) for @CamiMonte  intricate details which I missed.

### CODE REVIEW

File: models/db.js [hunk 1]
```diff
@@ -6,8 +6,7 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
-mongoose.connect(process.env.MONGOLAB_URI);
-//mongoose.connect("mongodb://localhost:27017/mongodb");
+mongoose.connect(process.env.MONGOLAB_URI || "mongodb://localhost:27017/mongodb");
```
Comment (matjames007) [inline models/db.js:9]: @thinkking and @nick22891 I don't think it is a good idea to default to localhost if the proper connection isn't found in the env variables.  If it doesn't find the correct params, this should fail.

File: public/javascript/controllers/user-ctrl.js [hunk 2]
```diff
@@ -123,6 +123,14 @@ angular.module('harvestv2')
             };
 
         }
```
Comment (matjames007) [inline public/javascript/controllers/user-ctrl.js:125]: @thinkking comments?

File: public/partials/admin-apps-activities.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<div ng-include src="'/partials/common/navigation.html'"></div>
```
Comment (matjames007) [inline public/partials/admin-apps-activities.html:1]: These two navigation files (`navigation.html` and `navigation-admin.html`) should be a part of the main ejs page.  No need for them to be continuously changes for every page that is loaded.  @thinkking

**Comment by matjames007** [comment]
@thinkking usually angular runs as a single page application.  From that single page the structure should be:

```
<navigation></navigation>
    <application-view>
        <!-- everything that is dynamic changes frequently -->
    </application-view>
<footer></footer>
```

This reduces the repetitive code that I am seeing in nearly every partials html file.  As a rule of thumb, try to have as few files and as few lines of code as possible to achieve the maximum feature set.

**Comment by matjames007** [comment]
@nick22891 please review with @thinkking to try and update this PR and figure out what the delays are.

**Comment by leechristophermurray** [comment]
@matjames007 This story was implemented on a branch worked on by @nick22891 So the pull request is no longer needed.

**Pull request closed without merging**